orgs/contributors.yml

```diff
@@ -212,6 +212,7 @@ orgs:
     - WeiQuan0605
     - weresch
     - winkingturtle-vmw
+    - wli
```

Comment:
You filed this PR and signed the CLA as @weili-broadcom. So you should add yourself as `weili-broadcom` and not as `wli`.